content/12-target-details.md

```diff
@@ -1945,6 +1945,52 @@ java -jar bin/Main.jar
 
 
 
+<!--label:target-jvm-->
+### JVM
+
+<!--subtoc-->
+
+<!--label:target-jvm-getting-started-->
+#### Getting started with Haxe/JVM
+
+To get started with Haxe/JVM, create a new folder and save this class as `Main.hx`.
+
+[code asset](assets/HelloWorld.hx)
+
+To compile Haxe to JVM bytecode we need two obvious prerequisites installed:
+
+* [hxjava haxelib](http://lib.haxe.org/p/hxjava). This is the support library for the Java backend of the Haxe compiler.
+* [JRE - Java Runtime Environment](https://java.com/download/).
+
+Run the following from the command line:
+
+```hxml
+haxe --jvm bin/Main.jar --main Main
+```
+
+Another possibility is to create and run (double-click) a file called `compile.hxml`. In this example the hxml-file should be in the same directory as the example class.
```

Comment:
> run (double-click)

This is not the case for all operating systems. AFAIK double-click is set up only in windows if Haxe was installed via the official installer.

Reply:
Ok. Left in a comment that some OS's let you double-click on a jar, though I don't think I've ever run a jar this way.